gpustack/gateway/__init__.py

```diff
@@ -242,12 +242,12 @@ def ext_auth_plugin(cfg: Config) -> Tuple[str, WasmPluginSpec]:
                 },
                 "endpoint": {
                     "path": "/token-auth",
-                    "request_method": "POST",
+                    "request_method": "GET",
                     "service_name": f"{registry.name}.{registry.type}",
                     "service_port": registry.port,
                 },
                 "endpoint_mode": "forward_auth",
-                "timeout": 1000,
+                "timeout": 3000,
```

Comment:
Is this millisecond? Do we want to make it configurable like https://github.com/gpustack/gpustack/blob/main/gpustack/config/envs.py#L12